index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Laboratory work 2</title>
+        <link rel="stylesheet" href="styles/styles.css">
+        <link rel="stylesheet" media="(max-width:640px)" href="styles/styles_640.css">
+        <link rel="stylesheet" media="(min-width:641px) and (max-width:980px)" href="styles/styles_640_980.css">
+        <link rel="stylesheet" media="(min-width:981px)" href="styles/styles_980.css">
+        <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    </head>
+    <body>
+        <header class="menu full-width">
+            <nav class="d-flex full-width">
+                <div class="small-screen-menu d-flex full-width">
+                    <span class="d-flex">Navigation</span>
+                    <div class="menu-button d-flex">
+                        <i class="fas fa-bars d-flex"></i>
+                    </div>
+                </div>
+                <div class="big-screen-menu d-flex full-width">
+                    <ul class="d-flex">
+                        <li>
+                            <a href="">PSD WEBSITES</a>
```

Comment:
It's okay, but could you please expand a tag clickable area to make it possible to click on li element?